pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -346,28 +347,33 @@ func (p *PriorityQueue) runEnqueuePlugins(ctx context.Context, pInfo *framework.
 		if s.IsSuccess() {
 			continue
 		}
-		if pInfo.UnschedulablePlugins == nil {
-			pInfo.UnschedulablePlugins = sets.NewString()
-		}
 		pInfo.UnschedulablePlugins.Insert(pl.Name())
 		metrics.UnschedulableReason(pl.Name(), pod.Spec.SchedulerName).Inc()
-		klog.InfoS("PreEnqueue failed", "pod", klog.KObj(pod), "plugin", pl.Name(), "error", s.AsError())
+		if s.Code() == framework.Error {
+			klog.ErrorS(s.AsError(), "Unexpected error running PreEnqueue plugin", "pod", klog.KObj(pod), "plugin", pl.Name())
+		} else {
+			klog.V(5).InfoS("Status after running PreEnqueue plugin", "pod", klog.KObj(pod), "plugin", pl.Name(), "status", s)
+		}
 		return false
 	}
 	return true
 }
 
+func (p *PriorityQueue) addToActiveQ(pInfo *framework.QueuedPodInfo) error {
+	if !p.PreEnqueue(pInfo) {
+		return errors.New("preEnqueue checks failed")
```

Comment:
but it is not an error if PreEnqueue returns false, it just means the pod is gated